crates/oxc_ast/src/ast/js.rs

```diff
@@ -1732,6 +1732,16 @@ pub struct ImportNamespaceSpecifier {
     pub local: BindingIdentifier,
 }
 
+impl GetSpan for ImportDeclarationSpecifier {
```

Comment:
This should be moved to crates/oxc_ast/src/span.rs to keep this file clean

Reply:
Done!